JS/Lesson22/tugba/style.css

```diff
@@ -0,0 +1,39 @@
+body {
+    font-family: Arial, sans-serif;
+}
+
+.accordion-container {
+    width: 60%;
+    max-width: 600px;
+}
+
+.accordion {
+    display: flex;
+    justify-content: center;
+    background-color: #f9f9f9;
+    color: #444;
+    cursor: pointer;
+    padding: 18px;
+    width: 80%;
+    text-align: left;
+    border-color:rgb(229, 229, 233);
+    outline: none;
+    transition: 0.4s;
+    border-radius: 10px;
```

Comment:
I think the button looks better without border but it's a personal preference, fell free to keep your styling.

My suggestion would be to remove `border-color` and `border-radius` from current code and instead add the following: 
```
border:none;
 border-radius: 10px 10px 0 0;
```